src/main/java/teammates/storage/api/FeedbackResponseCommentsDb.java

```diff
@@ -485,11 +490,12 @@ private Object getFeedbackResponseCommentEntity(String courseId, Date createdAt,
         return getCommentsWithoutDeletedEntity(feedbackResponseCommentList);
     }
     
-    private List<FeedbackResponseComment> getFeedbackResponseCommentEntityForSendingState(String courseId, String feedbackSessionName,
-            CommentSendingState state) {
+    private List<FeedbackResponseComment> getFeedbackResponseCommentEntityForSendingState(String courseId,
+            String feedbackSessionName, CommentSendingState state) {
```

Comment:
Try to (not compulsory) move the 2nd line to the right of the parent element. It's more intuitive when the child does not appear on the left of the parent element. e.g.
```
private List<FeedbackResponseComment> getFeedbackResponseCommentEntityForSendingState(String courseId,
                                              String feedbackSessionName, CommentSendingState state) {
```